app/src/main/java/org/fossasia/susi/ai/helper/Utils.kt

```diff
@@ -22,4 +24,26 @@ object Utils {
         return uri.toASCIIString()
     }
 
-}
+    fun toMd5Hash(email: String?): String {
+        var result = ""
+        try {
+            val md5 = MessageDigest.getInstance("MD5")
+            val md5HashBytes: Array<Byte> = md5.digest(email?.toByteArray()).toTypedArray()
+            result = byteArrayToString(md5HashBytes)
+        } catch (e: Exception) {
+            result = "error: ${e.message}"
```

Comment:
Result should never contain error message. Either throw an exception or return null string denoting there has been an error